x11-terms/terminator/terminator-1.92.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1 virtualx xdg-utils
+
+DESCRIPTION="Multiple GNOME terminals in one window"
+HOMEPAGE="https://github.com/gnome-terminator/terminator"
+SRC_URI="https://github.com/gnome-terminator/terminator/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="dbus +libnotify"
+
+RDEPEND="
+	>=dev-libs/glib-2.32:2
+	dev-libs/keybinder:3[introspection]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	>=x11-libs/gtk+-3.16:3
+	x11-libs/vte:2.91[introspection]
+	dbus? ( sys-apps/dbus )
+	libnotify? ( x11-libs/libnotify[introspection] )
+"
+BDEPEND="
+	dev-util/intltool
```

Comment:
Generally one needs
`test? ( ${RDEPEND} )` in `BDEPEND` for python packages. Also test needs to appear in `IUSE`

Reply:
Yeah, makes sense.